src/datasources/db/v2/database-migrator.service.spec.ts

```diff
@@ -68,7 +66,7 @@ describe('PostgresDatabaseService', () => {
       providers: [DatabaseMigrator],
     }).compile();
 
-    connection = postgresDataSourceMock;
+    connection = mockPostgresDataSource;
```

Comment:
This can be assigned when the variable is declared.

Reply:
Done https://github.com/safe-global/safe-client-gateway/pull/2084/commits/d879b9176acbbdeeb93f2a51942c5afc52149ec4